cdap-data-fabric/src/main/java/io/cdap/cdap/data2/dataset2/lib/table/leveldb/LevelDBTableCore.java

```diff
@@ -330,9 +393,33 @@ public void close() {
     return new ImmutablePair<>(rowBeingRead, map);
   }
 
+  /**
+   * Delete the cell at specified row and column with max version {@link KeyValue.LATEST_TIMESTAMP}.
+   */
+  public void delete(byte[] row, byte[] column) throws IOException {
```

Comment:
This call is pretty confusing. If I'd to see the call, I'd expect it to delete all versions, not just some special version. What is the use case?

Reply:
Used by caller who don't care about writing value to a particular version.  

For instance, preview manager would 
put (row, col, val) 
then
get(row, col, val) and delete(row, col) 

All of the above operate on a default version, which is max version. 

To delete all versions, method would better to be named deleteAllVersions() and we should have corresponding getAllVersions(). At the moment, there is no strong need for deleteAllVersions() and getAllVersions()

Reply:
Still please name it as `deleteDefaultVersion`. Usually when there are versions a delete call without specifying version removes all of it, not some "magic" version

Reply:
Done. Updated all APIs to [get|put|delete]DefaultVersion